openlr/helpers.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include "openlr/graph.hpp"
+
+#include "geometry/mercator.hpp"
+#include "geometry/point2d.hpp"
+
+#include "base/timer.hpp"
+
+#include <sstream>
+#include <string>
+#include <type_traits>
+
+namespace openlr
+{
+inline bool PointsAreClose(m2::PointD const & p1, m2::PointD const & p2)
+{
+  // Taken from features_road_graph.cpp.
+  // TODO(mgsergio): Avoid constant duplication.
+  double constexpr kMwmRoadCrossingRadiusMeters = 2.0;
```

Comment:
Why not to avoid constant duplication right now?

Reply:
Done.